_migration-assistant/migration-phases/planning-your-migration/adjusting-settings-during-migration.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+title: Adjusting settings during migration
+nav_order: 60
+parent: Planning your migration
+grand_parent: Migration phases
+redirect_from:
+  - /migration-assistant/migration-phases/adjusting-settings-during-migration/
+---
+
+# Adjusting settings during migration
+
+During a migration there is an opportunity change names and paths of index and data structures that are not modifiable after they are declared. This page provides a cookbook of different scenarios and templates that can be used to make these adjustments.
+
+### Rename an index
```

Comment:
Do we suspect that this will be a common requirement/ask from customers?  I suspect that most customers will either NOT want to rename any of their indices, or if they do, will have very specific reasons and requirements around doing that.

Is this guidance general enough that more than one customer (ideally, more than 10%) would benefit from it?  If not, I'd rather keep this in a set of internal runbooks for more targeted communications with users.

Reply:
I think rename is common ask and its a simple and useful example case.  The only request that is more popular is how to change the shard count - requiring more thoughtful messaging of the calculations that I would prefer not to start with.

Do you think we should have a different scenario we could do that instead of this one?